.prettierrc

```diff
@@ -2,5 +2,9 @@
   "semi": true,
   "singleQuote": true,
   "jsxSingleQuote": true,
-  "bracketSameLine": true
+  "bracketSameLine": true,
```

Comment:
Some of those prettier config maybe can be part of the eslint config. We should try to remove prettier if we can to avoid conflicts. If we can't tell me to check how to add it to the script we are going to run for code formatting.